doc/man3/EVP_chacha20.pod

```diff
@@ -22,7 +22,7 @@ The ChaCha20 stream cipher for EVP.
 =item EVP_chacha20()
 
 The ChaCha20 stream cipher. The key length is 256 bits, the IV is 128 bits long.
-The first 32 bits consists of a counter in little-endian order followed by a 96
+The first 64 bits consists of a counter in little-endian order followed by a 64
 bit nonce. For example a nonce of:
 
 000000000000000000000002
```

Comment:
You need to change this to 64 bits value as well and the value below too.

Reply:
Good catch.
Fixed.